tests/unit/mock-api-data/terms-cat.json

```diff
@@ -0,0 +1,342 @@
+[
+  {
+    "term": "cat",
```

Comment:
The data is easy to get from http://www.cortical.io/resources_apidocumentation.html